CHANGES.md

```diff
@@ -1,3 +1,11 @@
+# grmtools 0.6.2 (2020-03-22)
+
+* Fix two two Clippy warnings and suppress two others.
```

Comment:
Remove one `two`.

Reply:
Oops, well that's a very meta-typo! Fixed in https://github.com/softdevteam/grmtools/pull/173/commits/d80a0ea544829c5664703ca15ed5336a0ba7d6f0.